mmv1/third_party/terraform/services/accesscontextmanager/resource_access_context_manager_service_perimeter_dry_run_egress_policy_test.go

```diff
@@ -0,0 +1,156 @@
+package accesscontextmanager_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+// Since each test here is acting on the same organization and only one AccessPolicy
+// can exist, they need to be run serially. See AccessPolicy for the test runner.
+
+func testAccAccessContextManagerServicePerimeterDryRunEgressPolicy_basicTest(t *testing.T) {
+	// Multiple fine-grained resources
```

Comment:
What is happening here that involves multiple fine-grained resources? I only see a single egress policy?

Reply:
There are two instances of `google_access_context_manager_service_perimeter_dry_run_egress_policy` being created in the test.

Reply:
Is there any value in having two fine grained resources in the test or can I just remove one instead? This was essentially copied from the original resource that was written by the TF team so I was just following their footsteps.

Reply:
It's useful to see that they can both exist at the same time without interfering with each other.